lib/core/utils/aggregateChecks.js

```diff
@@ -29,6 +29,8 @@ axe.utils.aggregateChecks = function (nodeResOriginal) {
 
 		if (type === 'none') {
 			// For none, swap pass and fail outcomes.
+			// The none type is for checks that must not pass. So a pass actually
```

Comment:
I’m not completely sure about this comment.

Reply:
I suggest "none-type checks should pass when result is false rather than true."